eth/tools/fixtures/helpers.py

```diff
@@ -40,6 +40,7 @@
     BaseVM,
 )
 from eth.vm.forks import (
+    ConstantinopleVM,
```

Comment:
Some of these changes - extracted from [this gist](https://gist.github.com/veox/5874314b11cb731e69256c0bd451c88c), via issue #1562 - are already pending inclusion as part of https://github.com/cburgdorf/py-evm/pull/2.

Since you're not including the portion that enables more `Constantinople` tests, this change (and the other one from same file) would be unrelated to the rest of this PR.

Reply:
You're right, removed the Constantinople parts.